common/source/docs/common-airvolute-DroneCore-Suite.rst

```diff
@@ -0,0 +1,134 @@
+.. _common-airvolute-DroneCore-Suite:
+
+===========================
+Airvolute DroneCore.Suite
+===========================
+
+**DroneCore.Suite** is a one stop autopilot and flight computer solution for developers of advanced drone systems demanding high computing power and high level of modularity. 
+It is the "core" of the drone containing most of its electronics in a compact form and it provides wide options for software development of the the most challenging AI and vission based applications.
+
+.. image:: ../../../images/airvolute_droneCore.Suite_9233.jpg
+
+It consists of a control part called **DroneCore.Pilot** and power part called **DroneCore.Power**.
+
+
+**DroneCore.Pilot** - top board for control of the aircraft containing
+
+-  **Jetson Xavier NX**
+-  Cube flight controller
+-  Internal 5V power supply
+-  Power selector
+-  Peripheral connectors
+-  USB hub
+
+
+**DroneCore.Power** - bottom board with power electronic containing
+
+-  4 FOC UAVCAN ESCs - 20A permanent / 40A peak current
+-  Stacked configuration allows 8 ESCs
+-  Power sensor (battery voltage and current measurement)
+-  Analog inputs for battery cell monitoring
+-  SMBUS
+-  LED Driver for 4 x WS2812B strips
+
+
+
+
+DroneCore.Suite comes with both Cube and Xavier NX preconfigured and communicating together and able to control a quadcopter. Developers no longer need to solve basic communication and compatibility issues coming from connecting two different IT worlds. It They can start to concentrate on their application needs from the beginning.
+
+Accompanying documentation provides a fast learning curve to start using various peripherals to control the drone and create the desired application.
+Various software libraries (coming soon) further enrich the possibilities for the developer relieving him from complex configuration and programming tasks.
+
+DroneCore.Suite provides easy to use ROS based software stack supporting development of autonomous applications.
+
+Out of the box device is configured to fly a quadcopter (though flight controller calibration is needed anyway).
+
+---------------------------
+Top view
+---------------------------
+.. image:: ../../../images/airvolute_DCS_top_view_9219.jpg
+
+
+---------------------------
+Specifications:
+---------------------------
+
+**Mechanical parameters**
+
+-  Weight 247g - including Cube orange, Xavier NX, Xavier heatsink with
+   fan
+-  Dimensions 115 x 80 x 45mm
+-  Mount with 4 screws/spacers M3 109 x 74mm
+-  WiFi module
+-  WiFi antennas 2pcs
+
+**Electrical parameters**
+
+-  Power supply range: 12V – 35V (6S LiHV)
+-  Integrated DC/DC converter for control circuits
+-  Redundant power supply with power good monitoring for control unit
+-  Current protected peripheral connectors
+-  4 x FOC UAVCAN ESC 40A, featuring motor identification and motor
+   diagnostics
+-  Power sensor, SMBUS
+
+**Ardupilot (Cube) Connections**
+
+-  PPM input
```

Comment:
Only supports PPM RC?

Reply:
as input from Radio, the same way as Hex aero carrier boards do. What do you expect here?